src/main/java/com/pinterest/secor/reader/SecorKafkaMessageIterator.java

```diff
@@ -154,12 +153,20 @@ public void subscribe(RebalanceHandler handler, SecorConfig config) {
         }
     }
 
-
     private boolean getSkipZookeeperOffsetSeek(SecorConfig config) {
-
         String dualCommitEnabled = config.getDualCommitEnabled();
         String offsetStorage = config.getOffsetsStorage();
-        return offsetStorage.equals("kafka") && dualCommitEnabled.equals("true");
+        return offsetStorage.equals("kafka") && dualCommitEnabled.equals("false");
```

Comment:
this change is the interesting one.
Notice it determines if we are going to read from zk offset on new consumer after rebalance. https://github.com/remerge/secor/blob/c2d46a48ebbbbbd13d374e05fd540953094bbfa3/src/main/java/com/pinterest/secor/rebalance/SecorConsumerRebalanceListener.java#L53-L70